install/debian-installation-guide.md

```diff
@@ -0,0 +1,47 @@
+# Install Kata Containers on Debian
+
+These install instructions are based on the Ubuntu packages, but we need to install an updated version of `librbd1`.
+Tested only on Debian 9 - Stretch (x86_64).
```

Comment:
I'd remove this line (and put it in the commit message if you want).

Reply:
removed